.rubocop.yml

```diff
@@ -18,7 +18,7 @@ Layout/MultilineMethodCallIndentation:
   EnforcedStyle: indented
 
 Metrics/LineLength:
-  Max: 200  # Maximum line length from `rails new`
+  Max: 10000  # Maximum line length from `rails new`
```

Comment:
Do you think it is better to remove this rule rather than make it unreasonable? Just curious on your thoughts.

Reply:
We discussed and removing this causes an error so it isn't worth it for now.